src/roles/nuh-deploy/tasks/main.yml

```diff
@@ -128,6 +128,20 @@
         with_items:
           - "{{ external_interfaces }}"
 
+      - name: Copy proxy certificates over to secondary nuh
+        synchronize:
+          src: "{{ item }}"
+          dest: "{{ item }}"
+          mode: pull
+          private_key: "/root/.ssh/proxypeer"
+        delegate_to: "{{ groups['nuhs'][1] }}"
```

Comment:
Will this be an IndexError if there is only 1 NUH?

Reply:
Running a test now

Reply:
done, Ready for review